### ISSUE TITLE
no sysvinit script for couchdb

### ISSUE THREAD

**Issue by nledez** [cozy/couchdb-debian#1]
_From @jeekajoo on September 14, 2015 21:46_

Hi,

I'm using debian jessie in a LXC.
Official documentation advises to switch back to Sysvinit because of incompatility with systemd: https://wiki.debian.org/LXC#Incompatibility_with_systemd
This is what I did.
But it looks like there is no init script in your couchdb package: https://github.com/nledez/cozy-couchdb-debian. So it won't start and installs fails.. Too bad.
Am I wrong with my observation?

Thanks

_Copied from original issue: cozy/cozy-debian#17_

**Comment by ThibaudDauce** [comment]
Same problem here! :-)

For my friend Google (I had trouble finding this issue and the problem with systemd):

```
Creating UNIX users...
Generating CouchDB tokens...
Waiting for CouchDB to start... (1/20)
Waiting for CouchDB to start... (2/20)
Waiting for CouchDB to start... (3/20)
Waiting for CouchDB to start... (4/20)
Waiting for CouchDB to start... (5/20)
Waiting for CouchDB to start... (6/20)
Waiting for CouchDB to start... (7/20)
Waiting for CouchDB to start... (8/20)
Waiting for CouchDB to start... (9/20)
Waiting for CouchDB to start... (10/20)
Waiting for CouchDB to start... (11/20)
Waiting for CouchDB to start... (12/20)
Waiting for CouchDB to start... (13/20)
Waiting for CouchDB to start... (14/20)
Waiting for CouchDB to start... (15/20)
Waiting for CouchDB to start... (16/20)
Waiting for CouchDB to start... (17/20)
Waiting for CouchDB to start... (18/20)
Waiting for CouchDB to start... (19/20)
Waiting for CouchDB to start... (20/20)
Waiting for CouchDB to start... (21/20)
CouchDB is too long to start
dpkg: error processing package cozy-standalone (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 cozy-standalone
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Then after rerunning `cozy_management install_cozy`, no more CouchDB problem but:

``` bash
Generating CouchDB tokens...
Configuring Cozy Controller...

Creating config file /etc/supervisor/conf.d/cozy-controller.conf with new version
Restarted supervisord
Waiting for Cozy Controller to start... (1/20)
Waiting for Cozy Controller to start... (2/20)
Waiting for Cozy Controller to start... (3/20)
Waiting for Cozy Controller to start... (4/20)
Waiting for Cozy Controller to start... (5/20)
Waiting for Cozy Controller to start... (6/20)
Waiting for Cozy Controller to start... (7/20)
Waiting for Cozy Controller to start... (8/20)
Waiting for Cozy Controller to start... (9/20)
Waiting for Cozy Controller to start... (10/20)
Waiting for Cozy Controller to start... (11/20)
Waiting for Cozy Controller to start... (12/20)
Waiting for Cozy Controller to start... (13/20)
Waiting for Cozy Controller to start... (14/20)
Waiting for Cozy Controller to start... (15/20)
Waiting for Cozy Controller to start... (16/20)
Waiting for Cozy Controller to start... (17/20)
Waiting for Cozy Controller to start... (18/20)
Waiting for Cozy Controller to start... (19/20)
Waiting for Cozy Controller to start... (20/20)
Waiting for Cozy Controller to start... (21/20)
Cozy Controller is too long to start
dpkg: error processing package cozy-standalone (--configure):
 subprocess installed post-installation script returned error exit status 1
```

See also for french people https://forum.cozy.io/t/install-couchdb-refuse-de-se-lancer/703/2
« J'ai packagé avec systemd. Donc s'il n'est pas présent dans la VM cela ne fonctionne pas. »

**Comment by nledez** [comment]
Fixed with 4823d57c24c5d3bb407b3b17654940852f1b598c

**Comment by jeekajoo** [comment]
thanks
